src/Symfony/Component/ObjectMapper/Attribute/Map.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ObjectMapper\Attribute;
+
+/**
+ * Configures a class or a property to map to.
+ *
+ * @experimental
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
+ *
+ * @psalm-type CallableType =
```

Comment:
```suggestion
 * @psalm-type CallableType = string|callable(mixed $value, object $object): mixed
```

Reply:
actually @mtarld suggests that I removed that in favor of repeating types.

Reply:
Indeed, not all Symfony users are relying on psalm, therefore their IDE won't be able to recognize this type properly.